openwrt-openwisp-monitoring/files/monitoring.agent

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
 VERSION=0 #default value of version
+COLLECT=0
```

Comment:
wouldn't it be more readable if we had something like `MODE="collect"` and `MODE="send"`?